webanno/webanno
# #1806 - Display incoming and outgoing relations in the details panel

**What's in the PR**
- Display incoming/outgoing link/relations in the annotation detail editor panel

**How to test manually**
* Use an annotation to fill a slot. If you click on the annotation, you should see the slot owner linked in the detail panel
* Create a relation between two span annotations. If you click on either span, you should see the other linked in the detail panel.

**Automatic testing**

**Documentation**
